src/Model/BSONIterator.php

```diff
@@ -0,0 +1,117 @@
+<?php
+/*
+ * Copyright 2018 MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace MongoDB\Model;
+
+use MongoDB\BSON;
```

Comment:
This looks like a dead import, since `MongoDB\BSON` isn't used below. Was this done in anticipation of callingthe [BSON functions](http://php.net/manual/en/ref.bson.functions.php)?

PHP 5.6 does support [`use function`](https://wiki.php.net/rfc/use_function), although that won't work on 5.5, which we're still supporting. For now, I'd stick to calling `\MongoDB\BSON\toPHP()` and the like.